web/src/hooks/useApiRequest.tsx

```diff
@@ -29,8 +30,7 @@ export function useApiRequest<TResult>(args: UseApiRequestArgs<TResult>): UseApi
         setResponse(response);
       })
       .catch((err: Error) => {
-        console.error(err);
-        setErrorMessage(`Request failed: ${err.message}`);
+        axiosErrorHandlerWithCallback(err, () => setErrorMessage(`Request failed: ${err.message}`));
```

Comment:
Maybe it will be better to do this in the following way:
```typescript
.catch(defaultAxiosHandler({callback: () => setErrorMessage(`Request failed: ${err.message}`)}))

…

axiosErrorHandlerWithCallback = (opts) => {
  return (err: Error) => {
     if (!Axios.isCancel(err)) {
         console.log(err);
         if opts.callback { opts.callback() };
     }
  }  
}
```

If you don’t need a callback - just skip it.